README.md

```diff
@@ -1,2 +1,29 @@
 # SuperLists
 That's a test from the book `Test-Driven Development with Python` by H.J.W. Percival. The goal of this project is to learn TDD.
+
```

Comment:
We need to specify the requirement.
I am thinking about `virtualenvs`, `python`, `django` in specify the version.
##### Example:
    virtualenv==15.1.0
Did I miss someone?

Reply:
Done.